packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -247,37 +247,72 @@
     {{#each @model.TOGGLE_BUTTON_COLORS as |color|}}
       {{#each @model.TOGGLE_STATES as |state|}}
         <SG.Item @label="{{capitalize color}}">
-          <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @color={{color}} mock-state-value={{state}} />
+          {{#if (eq state "disabled")}}
+            <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @color={{color}} mock-state-value={{state}} disabled />
+          {{else}}
+            <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @color={{color}} mock-state-value={{state}} />
+          {{/if}}
         </SG.Item>
       {{/each}}
       <SG.Item @label="{{capitalize color}}">
         <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @isOpen={{true}} @color={{color}} />
       </SG.Item>
       {{#each @model.TOGGLE_STATES as |state|}}
         <SG.Item @label="With icon">
-          <Hds::Dropdown::Toggle::Button @icon="hexagon" @text="Lorem" @color={{color}} mock-state-value={{state}} />
+          {{#if (eq state "disabled")}}
+            <Hds::Dropdown::Toggle::Button
+              @icon="hexagon"
+              @text="Lorem"
+              @color={{color}}
+              mock-state-value={{state}}
+              disabled
+            />
+          {{else}}
+            <Hds::Dropdown::Toggle::Button @icon="hexagon" @text="Lorem" @color={{color}} mock-state-value={{state}} />
+          {{/if}}
         </SG.Item>
       {{/each}}
       <SG.Item @label="With icon">
         <Hds::Dropdown::Toggle::Button @icon="hexagon" @text="Lorem" @isOpen={{true}} @color={{color}} />
       </SG.Item>
       {{#each @model.TOGGLE_STATES as |state|}}
         <SG.Item @label="With count">
-          <Hds::Dropdown::Toggle::Button @count="12" @text="Lorem" @color={{color}} mock-state-value={{state}} />
+          {{#if (eq state "disabled")}}
+            <Hds::Dropdown::Toggle::Button
+              @count="12"
+              @text="Lorem"
+              @color={{color}}
+              mock-state-value={{state}}
+              disabled
+            />
+          {{else}}
+            <Hds::Dropdown::Toggle::Button @count="12" @text="Lorem" @color={{color}} mock-state-value={{state}} />
+          {{/if}}
         </SG.Item>
       {{/each}}
       <SG.Item @label="With count">
         <Hds::Dropdown::Toggle::Button @count="12" @text="Lorem" @isOpen={{true}} @color={{color}} />
       </SG.Item>
       {{#each @model.TOGGLE_STATES as |state|}}
         <SG.Item @label="With badge">
-          <Hds::Dropdown::Toggle::Button
-            @badge="Sit"
-            @badgeIcon="hexagon"
-            @text="Lorem"
-            @color={{color}}
-            mock-state-value={{state}}
-          />
+          {{#if (eq state "disabled")}}
+            <Hds::Dropdown::Toggle::Button
+              @badge="Sit"
+              @badgeIcon="hexagon"
+              @text="Lorem"
+              @color={{color}}
+              mock-state-value={{state}}
```

Comment:
I think you don't need to mock the disabled state, by using directly the `disabled` attribute
```suggestion
```